components/brave_new_tab_ui/components/default/editCards/style.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'brave-ui/theme'
+import palette from 'brave-ui/theme/colors'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
```

Comment:
Please can we use `<button>` for this?

Would need additional css:
```css
appearance: none;
margin: 0;
border: none;
width: 100%;
display: block;
```

Reply:
Done